### ISSUE TITLE
Failed to read label map file.

### ISSUE THREAD

**Issue by gzz1529657064** [eric612/MobileNet-YOLO#191]
Opened lmdb /home/gzz/data/VOCdevkit/MyDataSet/lmdb/MyDataSet_trainval_lmdb/
I1018 09:27:19.771530 17056 annotated_data_layer.cpp:78] output data size: 4,3,608,608
[libprotobuf ERROR google/protobuf/text_format.cc:274] Error parsing text-format caffe.YoloSegLabel: 4:15: Message type "caffe.YoloSegItem" has no field named "display_name".
F1018 09:27:19.771678 17056 annotated_data_layer.cpp:134] Check failed: ReadProtoFromTextFile(label_map_file_, &label_map_) Failed to read label map file.
*** Check failure stack trace: ***
    @     0x7fb222a095cd  google::LogMessage::Fail()
    @     0x7fb222a0b433  google::LogMessage::SendToLog()
    @     0x7fb222a0915b  google::LogMessage::Flush()
    @     0x7fb222a0be1e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fb223157650  caffe::AnnotatedDataLayer<>::DataLayerSetUp()
    @     0x7fb22318e58e  caffe::BasePrefetchingDataLayer<>::LayerSetUp()
    @     0x7fb22306d613  caffe::Net<>::Init()
    @     0x7fb22306fbce  caffe::Net<>::Net()
    @     0x7fb22324f7da  caffe::Solver<>::InitTrainNet()
    @     0x7fb22324fe15  caffe::Solver<>::Init()
    @     0x7fb22325010f  caffe::Solver<>::Solver()
    @     0x7fb223277297  caffe::Creator_RMSPropSolver<>()
    @           0x40be00  train()
    @           0x4087ce  main
    @     0x7fb221910830  __libc_start_main
    @           0x409029  _start
    @              (nil)  (unknown)
Aborted (core dumped)

@eric612 I meet this problem, How should I do?

**Comment by EvaneSnow** [comment]
hi @gzz1529657064  do you have solved the problem?

**Comment by gzz1529657064** [comment]
> hi @gzz1529657064  do you have solved the problem?

Yes, I use wrong prototxt, Now, I use models/yolov3/mobilenet_yolov3_train.prototxt and mobilenet_yolov3_test.prototxt. This prototxt data layer has only two top, data and label.
